iree/modules/strings/strings_module_test.cc

```diff
@@ -18,32 +18,50 @@
 #include "absl/strings/string_view.h"
 #include "iree/base/api.h"
 #include "iree/base/logging.h"
+#include "iree/hal/api.h"
+#include "iree/modules/hal/hal_module.h"
+#include "iree/modules/strings/api.h"
 #include "iree/modules/strings/strings_module_test_module.h"
 #include "iree/testing/gtest.h"
 #include "iree/vm/bytecode_module.h"
 #include "iree/vm/context.h"
 #include "iree/vm/instance.h"
 #include "iree/vm/module.h"
 #include "iree/vm/ref.h"
+#include "iree/vm/ref_cc.h"
 #include "iree/vm/stack.h"
 #include "iree/vm/types.h"
 
 using testing::internal::CaptureStdout;
 using testing::internal::GetCapturedStdout;
 
+namespace iree {
+
 namespace {
 
 class StringsModuleTest : public ::testing::Test {
  protected:
   virtual void SetUp() {
     IREE_CHECK_OK(iree_vm_instance_create(IREE_ALLOCATOR_SYSTEM, &instance_));
 
+    // Setup strings module:
     IREE_CHECK_OK(strings_module_register_types());
-
     IREE_CHECK_OK(
         strings_module_create(IREE_ALLOCATOR_SYSTEM, &strings_module_))
-        << "Native module failed to init";
+        << "Strings module failed to init";
+
+    // Setup hal module:
+    IREE_CHECK_OK(iree_hal_module_register_types());
+    iree_hal_driver_t* hal_driver = nullptr;
+    IREE_CHECK_OK(iree_hal_driver_registry_create_driver(
+        iree_make_cstring_view("vmla"), IREE_ALLOCATOR_SYSTEM, &hal_driver));
```

Comment:
why is vmla needed?

Reply:
We need some driver to test going from a HalBufferView to a StringTensor. It doesn't technically matter which one but I thought best to stay to VMLA so we get hardware independent support.